magda-storage-api/src/createApiRouter.ts

```diff
@@ -26,7 +26,27 @@ export default function createApiRouter(options: ApiRouterOptions) {
         return res.status(200).send("OK");
     });
 
-    // Download an object
+    /**
+     * @apiGroup Storage API
+     *
+     * @api {get} /v0/{bucket}/{fieldid} Request to download an object in {bucket} with name {fieldid}
+     *
+     * @apiDescription Downloads an object
+     *
+     * @apiParam (Request body) {string} bucket The name of the bucket under which the requested object is
+     * @apiParam (Request body) {string} fieldid The name of the object being requested
+     *
+     * @apiSuccess {string} status OK
```

Comment:
This comes out like

<img width="1016" alt="image" src="https://user-images.githubusercontent.com/900555/72479615-15ef6500-3849-11ea-876e-60ba0a6ebb01.png">

Which implies that `status` is a `string` returned in the body of the response.

I think what you want to do is something more like what authorization-api does - leave out the `@apiSuccess` bit and just keep the `@apiSuccessExample`.